content/get-haiku/r1beta4/release-notes.md

```diff
@@ -0,0 +1,189 @@
+
+### New NTFS driver
+
+The NTFS filesystem driver (which is a wrapper around the NTFS-3G library) was completely rewritten. The old one hearkened back to the BeOS days and had not kept up with development: it did not support any kind of caching, incorrectly handled file deletions in a way that caused kernel panics, did not support memory-mapping files (making `git` usage impossible on NTFS partitions), and had plenty of other issues besides.
+
+The new diver is much more idiomatic, supports the major layers of Haiku's common filesystem caching mechanisms, and resolves all known issues and major missing features the previous one was plagued with. Since it is not FUSE-based (like NTFS-3G is on Linux) but runs in kernel-mode, it is competitive for performance with the other fully native filesystem drivers.
+
+### AVIF translator
+
+There is now an AVIF image translator available by default, providing both read and write support for this new and advanced image format for all native Haiku applications.
+
+### USB-RNDIS tethering support
+
+A new driver has been added to support "RNDIS" ethernet tethering, which is the most common kind mobile phones use when selecting "USB tethering". It is enabled by default, and supports hot-plugging.
+
+### Bootloader
+
+32-bit EFI support has been added (previous releases only supported 64-bit EFI.) It is possible to boot a 64-bit Haiku installation from a 32-bit EFI loader.
+
+## Software ports
+
+### GTK applications!
+
+Thanks to (at first) the new X11 compatibility layer, and (now) the new Wayland compatibility layer (more details below), there is now a working GTK3 port for Haiku! Ports of **Inkscape**, **GIMP**, and more are now available for immediate install, and more GTK applications are being added to HaikuPorts as time goes on.
+
+One of the newly available GTK applications is **Epiphany** aka. **GNOME Web**, which is based on a very recent version of WebKitGTK. This provides an unfortunately non-native but largely functional web browser for Haiku for the first time in many years, with "just works" status for major websites like YouTube and others.
+
+### GNU Emacs
+
+<img src="/files/get-haiku/r1beta4/emacs.png" width="50%">
+
+Haiku has had terminal-based GNU Emacs for some time, but now (thanks to Emacs developers!) has a fully-upstreamed, polished-to-a-shine native GUI. You can install it from HaikuDepot already, or check out a development branch of Emacs and build it yourself.
+
+### WINE
+
+<img src="/files/get-haiku/r1beta4/wine.png">
+
+Thanks to community member & contributor X512, Haiku now has a port of WINE! Initially based on the X11 compatibility layer, it now has a fully native Haiku windowing & input backend. It is a bit inefficient performance-wise at present due to some limitations in Haiku, but that will likely improve with time as Haiku gains more I/O APIs.
```

Comment:
This is 64-bit only: it runs on 64-bit Haiku, and can only run 64bit windows apps. Making it a lot more limited than it sounds here.